core/domain/collection_domain.py

```diff
@@ -545,30 +761,75 @@ def _recursively_find_learned_skills(node):
 
     @classmethod
     def is_demo_collection_id(cls, collection_id):
-        """Whether the collection id is that of a demo collection."""
+        """Whether the collection id is that of a demo collection.
+
+        Args:
+            collection_id: str. The id of the collection.
+
+        Returs:
+            bool. True if the collection is a Demo else False.
+        """
         return collection_id in feconf.DEMO_COLLECTIONS
 
     @property
     def is_demo(self):
-        """Whether the collection is one of the demo collections."""
+        """Whether the collection is one of the demo collections.
+
+        Returs:
+            bool. True if the collection is a Demo else False.
+        """
         return self.is_demo_collection_id(self.id)
 
     def update_title(self, title):
+        """Updates the title of the collection.
+
+        Args:
+            title: str. The new title of the collection.
+        """
         self.title = title
 
     def update_category(self, category):
+        """Updates the category of the collection.
+
+        Args:
+            category: str. The new category of the collection.
+        """
         self.category = category
 
     def update_objective(self, objective):
+        """Updates the objective of the collection.
+
+        Args:
+            objective: str. The new objective of the collection.
+        """
         self.objective = objective
 
     def update_language_code(self, language_code):
+        """Updates the language code of the collection.
+
+        Args:
+            language_code: str. The new language code of the collection.
+        """
         self.language_code = language_code
 
     def update_tags(self, tags):
+        """Updates the tags of the collection.
+
+        Args:
+            tags: list(str). The new tags of the collection.
+        """
         self.tags = tags
 
     def _find_node(self, exploration_id):
+        """Returns the index of the collection node with the given exploration
+        id, or None if the exploration id is not in the nodes list.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+
+        Returns:
+            int. Index of node or None.
```

Comment:
int or None. The index of the corresponding node, or None if there is no such node.